packages/ts/react-form/test/useForm.spec.tsx

```diff
@@ -6,7 +6,7 @@ import chaiDom from 'chai-dom';
 import { useEffect, useState } from 'react';
 import sinon from 'sinon';
 import sinonChai from 'sinon-chai';
-import { useForm as _useForm, useFormArrayPart, useFormPart } from '../src/index.js';
+import { useForm as _useForm, useFormArrayPart, useFormPart } from '../src';
```

Comment:
I think we better not to use `../src` in test imports, but the exact file, in this case the `/index.js`:
```suggestion
import { useForm as _useForm, useFormArrayPart, useFormPart } from '../src/index.js';
```